session/bootstrap.go

```diff
@@ -853,6 +858,10 @@ func upgradeToVer34(s Session) {
 	doReentrantDDL(s, CreateOptRuleBlacklist)
 }
 
+func upgradeToVer35(s Session) {
+	mustExecute(s, "UPDATE HIGH_PRIORITY mysql.GLOBAL_VARIABLES SET VARIABLE_NAME='tidb_backoff_weight' WHERE VARIABLE_NAME='tidb_back_off_weight'")
```

Comment:
Do we need to update other variables? If we need to, I think we'd better do it in the same PR. Then we can reduce versions.

Reply:
No other name needs to be changed.